File: swift/Sources/Schemas/ReadyInputCapabilitiesItem.swift
import Foundation

public enum ReadyInputCapabilitiesItem: String, Codable, Hashable, CaseIterable, Sendable {
    case computeExecute = "compute.execute"
    case bashRun = "bash.run"
    case filesystemCheckpointV1 = "filesystem-checkpoint-v1"
    case conditionWaitsV1 = "condition-waits-v1"
}

File: ruby/lib/Sikaru/types/ready_input_capabilities_item.rb
# frozen_string_literal: true

module Sikaru
  module Types
    module ReadyInputCapabilitiesItem
      extend Sikaru::Internal::Types::Enum

      COMPUTE_EXECUTE = "compute.execute"
      BASH_RUN = "bash.run"
      FILESYSTEM_CHECKPOINT_V1 = "filesystem-checkpoint-v1"
      CONDITION_WAITS_V1 = "condition-waits-v1"
    end
  end
end
